pages/osx/date.md

```diff
@@ -0,0 +1,19 @@
+# date
+
+> Set or display the system date.
+
+- Display the date using the default locale:
+
+`date +"%c"`
+
+- Display the date in UTC and ISO 8601 format:
+
+`date -u +"%Y-%m-%dT%H:%M:%SZ"`
+
+- Display the number of seconds since the unix epoch:
+
+`date +%s`
+
+- Convert a unix expoch timestamp to a date:
```

Comment:
We usually try to include some words in the description that match the flags in the example. Can you update the description to contain a `j` and `r` ?

https://github.com/tldr-pages/tldr/issues/1018#issue-173587218